glossary.yml

```diff
@@ -4481,6 +4481,17 @@
       Phony targets are often used to store commonly-used commands in a [Makefile](#makefile).
 
 
+- slug: pip
+  ref:
+    - python
+  en:
+    term: "Pip Install Packages"
+    acronym: "PIP"
+    def: >
+      The standard [package manager](#package_manager) for [Python](#python). pip Enables the download and installation
+      of Python packages not included in the standard library.
```

Comment:
```suggestion
      The standard [package manager](#package_manager) for [Python](#python). pip enables
      the download and installation of Python packages not included in the standard library.
```


We try to keep lines from getting too wide—no set character limit, but use some longer entries in the file to see where to put line breaks; but don't split words.

Reply:
Also, `enables` didn't need to be capitalised.